zipline/assets/_assets.pyx

```diff
@@ -21,10 +21,6 @@ cimport cython
 import numpy as np
 cimport numpy as np
 
-cdef enum AssetType:
-    EQUITY = 1
-    FUTURE = 2
-
```

Comment:
@llllllllll I removed the enum and converted everything to isinstance() checks, as we discussed.